src/routes/local.ts

```diff
@@ -15,27 +19,33 @@ const PROPS_TO_SEND_IN_RESPOSNE = [
   "purpose",
 ];
 
-router.get("/two-line-elements/:id", (req, res) => {
-  if (req.params.id === undefined) return res.sendStatus(400);
-  n2yo
-    .getTwoLineElements(req.params.id)
-    .then((result) => res.json(result))
-    .catch(() => res.sendStatus(400));
-});
+router.get("/two-line-elements/:id",
+  (req, res, next) => satelliteIdRouteValidation(req, res, next),
+  (req: RequestWithID, res) => {
```

Comment:
`GetTwoLinesElementsRequest` perhaps? Same for the one with name

Reply:
In general if you have a variable that tells you literally the same thing than the value of the said variable it's a room for improvement. 
I.e. `const RED = 'red'` is kinda useless, because you can easily see that it's red
But 
`const HEADER_COLOR = 'red'`
Tells you much more. You know that it's red AND that we use it as a color of a header.